app/lib/local_links_manager/import/publishing_api_importer.rb

```diff
@@ -82,7 +82,7 @@ def publishing_api_response
       def local_transaction_hash(parsed_result)
         local_transaction = {}
         local_transaction["title"] = parsed_result["title"]
-        local_transaction["slug"] = parsed_result["base_path"][1..-1]
+        local_transaction["slug"] = parsed_result["base_path"][1..]
```

Comment:
This feels less readable, but its autocorrected so won't argue!

Reply:
Yup, that's a fight to take up with Rubocop 😁